modules/accredited_representatives/spec/requests/v0/power_of_attorney_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe AccreditedRepresentatives::V0::PowerOfAttorneyController, type: :request do
+  include RequestHelper
```

Comment:
@gabezurita Is this [handled automatically](https://github.com/department-of-veterans-affairs/vets-api/blob/7ea2af0020ab8f2b4309a59d79c88b2b66031b4b/spec/rails_helper.rb#L171) for all request specs?